[#342] Add guide on how to connect Rasa and fix webhook API (#423)
airyhq/airy: docs/docs/sources/chat-plugin.md

#### Send message

> You have to set the `token` obtained on the [authorization endpoint](#authenticating-web-users) as an `Authorization`

Reviewer: ```suggestion
You must set the `token` obtained on the [authorization endpoint](#authenticating-web-users) as an `Authorization`
```
